server/server.js

```diff
@@ -1,7 +1,13 @@
+import Database from "better-sqlite3";
 import { serve } from "@hono/node-server";
 import { Hono } from "hono";
+import queries from "./queries.js";
 
 const app = new Hono();
+const db = new Database("database.db");
+
+db.prepare(queries.Users.createTable).run();
+db.prepare(queries.DrinkingRecords.createTable).run();
```

Comment:
```suggestion
const db = new Database("database.db");

func migrate(db) {
  db.prepare(queries.Users.createTable).run();
  db.prepare(queries.DrinkingRecords.createTable).run();
}
```

こうするのがいいと思っていて、最後の `serve` の前に `migrate` するイメージ